L3-L4-OOP/L3-Zad.2-Movie/src/main/java/pl/movie/Movie.java

```diff
@@ -0,0 +1,47 @@
+package pl.movie;
+
+public class Movie {
+    private String title;
+    private String directorName;
+    private int featureLength;
+    private int yearOfProduction;
+
+    public Movie(String title, String directorName, int featureLength, int yearProduction) {
+        this.title = title;
+        this.directorName = directorName;
+        this.featureLength = featureLength;
+        this.yearOfProduction = yearProduction;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getDirectorName() {
+        return directorName;
+    }
+
+    public void setDirectorName(String directorName) {
+        this.directorName = directorName;
+    }
+
+    public int getFeatureLength() {
+        return featureLength;
+    }
+
+    public void setFeatureLength(int featureLength) {
+        this.featureLength = featureLength;
```

Comment:
nic w treści zadania nie ma o możliwości zmiany tych danych jak titel/directorname etc. Wystarczy, że w konstruktorze ustawimy te dane. Nie generuj od razu setterów dla wszystkich pól - to zła praktyka. Udostępniajmy tylko to co faktycznie jest wykorzystywane.